imgix/strapi-plugin-imgix
# Feat/library sync

## Issue
When installing a plugin over an existing Strapi instance, the Media Library resources are not initialized with imgix plugin.

## Solution
Introducing **Sync Media Library** and **Desync Media Library** administration actions.

<img width="965" alt="Screenshot 2024-06-24 at 11 31 49" src="https://github.com/imgix/strapi-plugin-imgix/assets/11504214/decca950-456e-4d7e-8a07-be883b8a483f">

## Tests
1. Setup plugin configuration
2. Sync Media Library
3. See that all resources are working with imgix overwrite
4. Desync Media Library
5. See that all resources are working with your native settings
